src/pybind/mgr/dashboard/frontend/src/app/ceph/block/rbd-list/rbd-list.component.ts

```diff
@@ -272,39 +276,41 @@ export class RbdListComponent extends ListWithDetails implements OnInit {
         pipe: this.dimlessBinaryPipe
       },
       {
-        name: $localize`Objects`,
-        prop: 'num_objs',
-        flexGrow: 1,
-        cellClass: 'text-right',
-        sortable: false,
-        pipe: this.dimlessPipe
-      },
-      {
-        name: $localize`Object size`,
-        prop: 'obj_size',
-        flexGrow: 1,
-        cellClass: 'text-right',
-        sortable: false,
-        pipe: this.dimlessBinaryPipe
-      },
-      {
-        name: $localize`Provisioned`,
+        name: $localize`Used`,
         prop: 'disk_usage',
         cellClass: 'text-center',
```

Comment:
let's be consistent throughtout!
```suggestion
        cellClass: 'text-right',
```